editor/build.gradle

```diff
@@ -4,7 +4,7 @@ dependencies {
     api "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"
     api "com.badlogicgames.gdx:gdx-freetype-platform:$gdxVersion:natives-desktop"
     implementation "info.debatty:java-string-similarity:1.2.1"
-
+    compile "com.esotericsoftware.spine:spine-libgdx:3.8.55.1"
```

Comment:
Should probably be implementation, and same for talos-libgdx.

Reply:
ok will change